src/sagas/server-configs.js

```diff
@@ -14,16 +14,17 @@ import {
 } from '_/actions'
 
 import { callExternalAction } from './utils'
-import { isNumber } from '_/utils'
 
 export function* fetchServerConfiguredValues () {
   const optionVersion = yield select(state => {
     const ver = state.config.get('oVirtApiVersion')
-    return `${ver.get('major')}.${ver.get('minor')}`
+    return (ver.get('major') === 4 && ver.get('minor') === 4)
+      ? `4.${ver.get('build')}`
```

Comment:
@sjd78 
do we want to keep this as temp fix? as discussed build number does not match max cluster number

Reply:
I think it's ok as a better quick fix than what is in there now.  

We will need to better address the engine-config value for the values that have more than the 'general' version per cluster level in near future.

Reply:
For API version lower then 4.4.5 it will query wrong cluster version (i.e. version 4.4.3 it will use cluster level 4.3).
Do we need to restrict it via API check?

Reply:
I believe the real answer is that the version number here is NOT the API version or the engine version but the cluster level.  More work is needed to support cluster version -> config value for the version -> arch -> value.